### PR TITLE
fix: sync with example project

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Syncs index.html and README.md with the example project at https://curse-arrow.glitch.me/.

Closes: https://github.com/freeCodeCamp/freeCodeCamp/issues/34626 based on comments @moT01 left in https://github.com/freeCodeCamp/boilerplate-project-timestamp/pull/19

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Would you mind taking a look at this when you have some time @ojeytonwilliams?

File: views/index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h3>Example Output:</h3>
       </div>
       <div class="footer">
         <p>
-          by <a href="http://www.freecodecamp.com/">freeCodeCamp</a> | <a href="https://hyperdev.com/#!/remix/FCC_API_timestamp_bplate/8338be2-a353-4892-8a0d-6295e4f12eb5">Click here to Remix</a>
+          by <a href="http://www.freecodecamp.com/">freeCodeCamp</a>
```
Comment (ojeytonwilliams) [inline views/index.html:44]:
```suggestion
          by <a href="http://www.freecodecamp.org/">freeCodeCamp</a>
```
Might as well go directly.

File: views/index.html [hunk 2]
```diff
@@ -6,14 +6,14 @@
       <title>Timestamp Microservice</title>
       <link rel="shortcut icon" href="https://cdn.hyperdev.com/us-east-1%3A52a203ff-088b-420f-81be-45bf559d01b1%2Ffavicon.ico" type="image/x-icon"/>
       <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css">
-      <link href="style.css" rel="stylesheet" type="text/css">
+      <link href="/public/style.css" rel="stylesheet" type="text/css">
```
Comment (ojeytonwilliams) [inline views/index.html:9]:
```suggestion
      <link href="style.css" rel="stylesheet" type="text/css">
```
It seems Glitch puts everything in the `/public` dir into the `/` route.

File: views/index.html (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply views/index.html:44]: Shoot, thank you for catching that. Definitely something that needs to be fixed on the working example project, too.

File: views/index.html (hunk 2 above)
Comment (scissorsneedfoodtoo) [reply views/index.html:9]: I tried this very quickly on my remix of this project and none of the styling was applied when I opened the project in another window. Maybe it's the way we're serving the `/public` directory in `server.js`?:

```js
app.use('/public', express.static(process.cwd() + '/public'));
```

Comment (ojeytonwilliams) [reply views/index.html:9]: My bad, I thought that was Glitch specific thing!

Where's 
```js
app.use('/public', express.static(process.cwd() + '/public'));
```
coming from?  I can't find that anywhere and the PR has this:
https://github.com/freeCodeCamp/boilerplate-project-timestamp/blob/2e7849ad37716fbffdaacc61e60716dfc69584b2/server.js#L14

Which does what I thought Glitch was doing, namely mount everything inside `/public` at the root of the site.

Here's a project which I started by cloning this PR and then added `href="style.css"`: https://glitch.com/~succinct-clover-aardwolf.

Comment (scissorsneedfoodtoo) [reply views/index.html:9]: @ojeytonwilliams, no problem at all! I'm still trying to wrap my head around all of this myself. Will be nice to get off Glitch entirely so there's not that extra layer of pastel colored complexity.

Thank you for pointing this out -- it's yet another place where the solution and boilerplate have somehow merged over the years. (Insert copy of your profile picture).

Mounting everything in the `/public` directory to the root is much easier and cleaner. I'll go in and update the PR now.

**Comment by scissorsneedfoodtoo** [comment]
Thank you for your help reviewing this @ojeytonwilliams, and thanks for the merge @moT01.

Yes, I agree, there's really no point to having the boilerplates on Glitch. It's just another thing to update, and I doubt that people are searching for the project boilerplates on Glitch. They're more than likely using that link or making a fork of this repo. 

I'll do a quick audit to make sure we don't have links to the boilerplates anywhere in the main repo before removing them from Glitch.

**Pull request merged**